proposals/0191-enable-transaction-loading-failure-fees.md

```diff
@@ -0,0 +1,106 @@
+---
+simd: '0191'
+title: Relax Transaction Loading Constraints
+authors:
+  - Andrew Fitzgerald (Anza)
+category: Standard
+type: Core
+status: Review
+created: 2024-11-06
+feature: PaymEPK2oqwT9TXAVfadjztH2H6KfLEB9Hhd5Q5frvP
```

Comment:
Let's link to the github issue instead: https://github.com/anza-xyz/agave/issues/3244

Reply:
Added link with feature key as well